### DISCUSSION TITLE
电脑部署，局域网内的电视KODI无法访问

### DISCUSSION THREAD

**Discussion by xinxinenjoy** [AlistGo/alist#2777] [Q&A] [answered]
WIN10系统部署好了Alist，可以正常访问，也能通过raidrive挂载。
可是同局域网内的电视无法使用KODI通过webdav来访问网盘内容特来请教。
IP是挂载电脑的IP，端口5244，账号密码为Alist的

**Comment by xinxinenjoy** [comment, marked as answer]
已经自行解决，不知道有没有参考价值
1.防火墙添加alist并允许（这时候还没效果）
2.防火墙里的高级设置，入站规则，找到alist全部启用。（搞定）

**Reply by ds3szy** [reply]
老哥我用你的方法尝试了下，情况和你一样本机可以访问，但是我同wifi下的手机和其他电脑都无法访问
![0464BA2D](https://user-images.githubusercontent.com/104285807/221129529-27b1e89a-76d9-491e-8fa0-2896113305f0.png)

**Comment by aurumple** [comment]
搞定了，新建入站和出站规则，添加alist.exe，允许所有通信，手机就能访问了
